packages/account/src/Components/reset-trading-password-modal/reset-trading-password-modal.tsx

```diff
@@ -179,13 +191,12 @@ const ResetTradingPassword = ({ setDialogTitleFunc, toggleResetTradingPasswordMo
     );
 };
 
-ResetTradingPassword.propTypes = {
-    is_dxtrade_allowed: PropTypes.bool,
-    setDialogTitleFunc: PropTypes.func,
-    toggleResetTradingPasswordModal: PropTypes.func,
-    verification_code: PropTypes.string,
-    platform: PropTypes.string,
-};
+export type TResetTradingPasswordModal = {
```

Comment:
@suisin-deriv No need to export the type, You can use the `React.ComponentProps` generic if needed.

Reply:
Oriteee let me edit it 🙏